src/settings/Settings.js

```diff
@@ -224,6 +224,12 @@ export const SETTINGS = {
         default: true,
         invertedSettingName: 'dontSendTypingNotifications',
     },
+    "showTypingNotifications": {
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        displayName: _td("Show typing notifications"),
+        default: true,
+        invertedSettingName: 'dontShowTypingNotifications',
```

Comment:
This shouldn't be here - there's no legacy setting to maintain.

Reply:
Are you referring solely to line 231? It's a bit unclear from how github marked it.

Reply:
yes, 231. The `invertedSettingName` is a backwards compatibility thing, but there's nothing to be backwards compatible with here.